LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/OTAAHelper/OTAAKeysGenerator.cs

```diff
@@ -10,31 +10,30 @@ namespace LoRaTools
 
     public static class OTAAKeysGenerator
     {
-        public static DevAddr GetNwkId(uint netId)
+        public static DevAddr GetNwkId(NetId netId)
```

Comment:
This method seems to be only used in one place and doesn't have much to do with OTAA key generation. Why not consider inlining as part of this PR?

Reply:
Done with 5b4bb44c5fccb14409ab6d5e1c264ef9fe5d4073